ingest_python_scripts/nara/launch_nara.py

```
@@ -0,0 +1,365 @@
+import json
+import re
+import subprocess
+import sys
+import time
+from datetime import datetime
+
+# ---------- config ----------
+REGION         = "us-east-1"
+def _load_dotenv():
+    cfg = {}
+    env_file = os.path.normpath(
```

Ruff (0.15.12)

[error] Undefined name `os`

(F821)